layer/clients/executor_service.py

```diff
@@ -42,3 +48,21 @@ def get_function_download_path(
         return self._service.GetFunctionDownloadPath(
             request=request
         ).function_download_path
+
+    def get_or_create_cluster(
+        self, account_id: uuid.UUID, language_version: Tuple[int, int, int]
+    ) -> str:
+        idle_scale_down_duration_td = timedelta(minutes=10)
+        idle_shut_down_duration_td = timedelta(hours=1)
+        idle_scale_down_duration = Duration()
+        idle_scale_down_duration.FromTimedelta(td=idle_scale_down_duration_td)
+        idle_shut_down_duration = Duration()
+        idle_shut_down_duration.FromTimedelta(td=idle_shut_down_duration_td)
+        # TODO(emin): add account_id and language_version as parameter to below
+        request = GetOrCreateClusterRequest(
+            idle_scale_down_duration=idle_scale_down_duration,
+            idle_shut_down_duration=idle_shut_down_duration,
+            max_number_of_workers=10,  # TODO(emin): remove this parameter
+            worker_fabric="",  # TODO(emin): remove this parameter as it should be defined by the function executable for each
```

Comment:
Until we figure out how do we make them configurable and expose it to the user we can keep them as defaults.